winlogbeat/beater/winlogbeat.go

```diff
@@ -114,8 +106,8 @@ func (eb *Winlogbeat) Setup(b *beat.Beat) error {
 
 	// Create the event logs. This will validate the event log specific
 	// configuration.
-	eb.eventLogs = make([]eventlog.EventLog, 0, len(eb.config.Winlogbeat.EventLogs))
-	for _, config := range eb.config.Winlogbeat.EventLogs {
+	eb.eventLogs = make([]eventlog.EventLog, 0, len(config.EventLogs))
+	for _, config := range config.EventLogs {
 		eventLog, err := eventlog.New(config)
```

Comment:
There is a lot of useful configuration validation performed as part of this `eventlog.New` method. It would be best if most of what is in this `Setup` method was invoked as part of `-configtest` (with the exception of the part where it launches an http server).